tests/unit/Administration/Login/Config/LoginConfigServiceTest.php

```diff
@@ -0,0 +1,313 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Tests\Unit\Administration\Login\Config;
+
+use PHPUnit\Framework\Attributes\CoversClass;
+use PHPUnit\Framework\Attributes\DataProvider;
+use PHPUnit\Framework\TestCase;
+use Shopware\Administration\Login\Config\LoginConfigService;
+use Shopware\Administration\Login\Exception\LoginException;
+use Shopware\Core\Framework\Log\Package;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * @internal
+ */
+#[Package('after-sales')]
+#[CoversClass(LoginConfigService::class)]
+class LoginConfigServiceTest extends TestCase
+{
+    public function testGetConfigWithEmptyRawConfig(): void
+    {
+        $configService = new LoginConfigService(
+            [
+                'use_default' => true,
+                'client_id' => 'clientId',
+                'client_secret' => 'clientSecret',
+                'redirect_uri' => 'redirectUri',
+                'base_url' => 'baseUrl',
```

Comment:
Shouldn't this array be empty for it to return `null`?

Reply:
What happend here? :thinking:  IDK. Yes, the rawConfig should be empty